libmscore/arpeggio.cpp

```diff
@@ -434,6 +440,8 @@ QVariant Arpeggio::getProperty(Pid propertyId) const
 bool Arpeggio::setProperty(Pid propertyId, const QVariant& val)
       {
       switch(propertyId) {
+            case Pid::TIME_STRETCH:
+                  setStretch(val.toDouble());
```

Comment:
is the "Fall THrough" intended here? Of so better add a `/* FALLTHROUGH */`, to a) document and b) avoid a compiler warning

Reply:
Actually I believe this is a bug and a `break;` is missing here. For once a valid compiler warning ;-)